### PR TITLE
Fixed overlapping notifications

### PR DESCRIPTION
**Pull request by rupato-deriv**

### CODE REVIEW

File: static/css/bot.scss [hunk 1]
```diff
@@ -56,6 +56,37 @@ body {
     #blocklyArea {
         height: 86%;
     }
+
+}
+@media (max-width: 768px){
```
Comment (salarhfz-fs) [inline static/css/bot.scss:61]: Can not use `mobile mixin` here?

Comment (rupato-deriv) [reply static/css/bot.scss:61]: I believe yes, we can use that.
Let me update the PR

**Review by mehrzad-fs — APPROVED** [review]

**Review by salarhfz-fs — APPROVED** [review]

**Review by sara-fs — APPROVED** [review]

File: src/botPage/view/logger.js [hunk 2]
```diff
@@ -18,8 +18,9 @@ const log = (type, ...args) => {
 
 const notify = ({ className, message, position = 'left', sound = 'silent' }) => {
     log(className, message);
+    const pos = isMobile() ? 'right' : position;
```
Comment (salarhfz-fs) [inline src/botPage/view/logger.js:21]: You can do it in the props and remove this line:
`position =isMobile() ? 'right' : 'left'`

Comment (mehrzad-fs) [reply src/botPage/view/logger.js:21]: Not a good idea to add pos here, you should pass it as a props

**Comment by mehrzad-fs** [comment]
please merge your PR to master, and resolve conflcits

**Review by salarhfz-fs — APPROVED** [review]

**Review by sara-fs — APPROVED** [review]

**Review by mehrzad-fs — APPROVED** [review]

**Review by sara-fs — APPROVED** [review]

**Review by mehrzad-fs — APPROVED** [review]

**Review by sara-fs — COMMENTED** [review]
In your test link I still can see the overlapping. I used the XML file attached to the card for testing.

**Pull request merged**